i2cdrivers/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
 dependencies {
     //compileOnly fileTree(include: ['*.aar'], dir: '../libs')
     compileOnly 'org.openftc:robotcore-stock:5.1'
+    implementation 'com.android.support:support-compat:28.0.0'
```

Comment:
The new SDK versions use androidx, so having appcompat as a dependency here will cause a compile time conflict. Can you change this to `compileOnly 'androidx.appcompat:appcompat:1.2.0'`?

You can then use this import: `import androidx.annotation.ColorInt;`

Reply:
I'll make this change.   Good call.